src/ds5/ds5-thermal-monitor.h

```diff
@@ -2,11 +2,41 @@
 // Copyright(c) 2021 Intel Corporation. All Rights Reserved.
 #pragma once
 
+#include "option.h"
 #include "sensor.h"
 #include "device-calibration.h"
 
 namespace librealsense
 {
+    template<typename T>
+    class protected_xu_option : public uvc_xu_option<T>
```

Comment:
Needs description

Reply:
Shouldn't this go with `uvc_xu_option`? Why here?

Reply:
Moved to the base header file.
Comment added